main.py

```diff
@@ -42,7 +42,7 @@ def show_frame(self, controller):
             self.frame[controller] = frame = controller(self.container, self)
             frame.grid(row=0, column=0, sticky="nsew")
         frame = self.frame[controller]
-        frame.tkraise()
+        frame.CTkraise()
```

Comment:
Please check that `customtkinter.CTkFrame` has this function.

Reply:
It does not unfortunately, I was getting errors and hoping that would fix it. But no luck.

Reply:
👍 Please revert that commit.